.github/workflows/ci.yml

```diff
@@ -24,5 +24,5 @@ jobs:
         run: |
           cargo build --no-default-features
           cargo test --no-default-features
-          cargo build --all-features
-          cargo test --all-features
\ No newline at end of file
+          cargo build --features unic --features derive --features card
+          cargo test  --features unic --features derive --features card
```

Comment:
What's the difference between those diffs?

Reply:
Can you add a nightly CI run that enables all the features? Otherwise that feature is never tested